### PR TITLE
(feature) putting links to SAV back in

### PR DESCRIPTION
**Pull request by appility**

### CODE REVIEW

File: app/views/enquiry_response_mailer/_not_for_third_party.html.haml [hunk 1]
```diff
@@ -18,6 +18,10 @@
 
 %p If you are representing a UK company please tell your client they must submit a proposal directly.
 
+%h3 Review your proposal and addtional recommendations
```
Comment (agiamas) [inline app/views/enquiry_response_mailer/_not_for_third_party.html.haml]: typo

File: app/views/enquiry_response_mailer/_not_right_for_opportunity.html.haml [hunk 2]
```diff
@@ -16,6 +16,10 @@
       %p Your proposal does not meet the criteria for this opportunity.
 
       = render 'enquiry_response_mailer/editable_content', enquiry_response: @enquiry_response
+      
+      %h3 Review your proposal and addtional recommendations
```
Comment (agiamas) [inline app/views/enquiry_response_mailer/_not_right_for_opportunity.html.haml]: same typo

File: app/views/enquiry_response_mailer/_not_right_for_opportunity.html.haml [hunk 3]
```diff
@@ -16,6 +16,10 @@
       %p Your proposal does not meet the criteria for this opportunity.
 
       = render 'enquiry_response_mailer/editable_content', enquiry_response: @enquiry_response
+      
+      %h3 Review your proposal and addtional recommendations 
+
+      %p= link_to "#{Figaro.env.DOMAIN}/dashboard/enquiries/#{@enquiry_response.enquiry_id}", "#{Figaro.env.DOMAIN}/dashboard/enquiries/#{@enquiry_response.enquiry_id}", { :class=>"link" }
```
Comment (agiamas) [inline app/views/enquiry_response_mailer/_not_right_for_opportunity.html.haml]: { class: 'link' } is better for rubocop and ruby 1.9+ code style

File: app/views/enquiry_response_mailer/_not_uk_registered.html.haml [hunk 4]
```diff
@@ -17,6 +17,10 @@
 
 %p If your company is registered in the UK, please make sure you include all the relevant company details - company name, UK address and phone number - next time you submit a proposal. 
 
+%h3 Review your proposal and addtional recommendations
```
Comment (agiamas) [inline app/views/enquiry_response_mailer/_not_uk_registered.html.haml]: same typo

File: app/views/enquiry_response_mailer/_not_uk_registered.html.haml [hunk 5]
```diff
@@ -17,6 +17,10 @@
 
 %p If your company is registered in the UK, please make sure you include all the relevant company details - company name, UK address and phone number - next time you submit a proposal. 
 
+%h3 Review your proposal and addtional recommendations 
+
+%p= link_to "#{Figaro.env.DOMAIN}/dashboard/enquiries/#{@enquiry_response.enquiry_id}", "#{Figaro.env.DOMAIN}/dashboard/enquiries/#{@enquiry_response.enquiry_id}", { :class=>"link" }
```
Comment (agiamas) [inline app/views/enquiry_response_mailer/_not_uk_registered.html.haml]: same style

File: app/views/enquiry_response_mailer/_right_for_opportunity.html.haml [hunk 6]
```diff
@@ -19,6 +17,12 @@
 
       %p Although you meet the criteria for this opportunity, we cannot guarantee that you will win the business. If you have any questions about this opportunity or our market in general, please do not hesitate to contact us.
 
+      %h3 Review your proposal and addtional recommendations
```
Comment (agiamas) [inline app/views/enquiry_response_mailer/_right_for_opportunity.html.haml]: same typo

**Comment by appility** [comment]
Thanks for the catch, amended now.

**Pull request merged**